### ISSUE TITLE
Project: License Plate Lookup

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#11448]
A webpage where the public can go to look up a license plate (LP) to see if it is permitted for the Residential Parking Permit zone or not.

## Background

Currently, there is no solution in place for residents to be able to check and verify if vehicles parked on their streets have been permitted or not. Most concerns are addressed by the residents contacting 3-1-1 or parking enforcement for assistance.

With the department transitioning to a digital permitting process for the RPP program, residents have requested a resource to be provided to them to assist them with monitoring vehicles parking in their particular zones. 

Previously, physical decals and hangtags were issued, which were visible for all residents to see. In an effort to alleviate their concerns, and assist in mitigating unnecessary calls to 3-1-1, or enforcement. The hope is that this platform will provide residents with an easy solution that they can access. 

## Scope & Deliverables

Citywide public application that is accessible on mobile and desktop.

## Desired Outcomes

Success occurs when residents are able to enter an LP into a search bar, and the search result will list the LP as either "Permitted" or "Not Permitted" and zone color.

## Timebox

February-May 2023. Deliver before renewal for Orange Zone starts in June.

## Concepts and references

Helpful links; e.g. other solutions, screenshots of Slack convos, customer’s original request, anything too long for the one-pager that can exist as a separate doc. Folder in Drive where project assets are stored.

[Current Knack App]( https://atd.knack.com/parking#rpp/)
[City of Austin RPP website where to include lookup](https://www.austintexas.gov/page/purchase-permits-residential-parking) 

![eligibility form.png](https://images.zenhubusercontent.com/5d68810b05d7792664be403f/2affa4e3-fc66-410b-850a-f7fc63b16394)

---
This Github issue represents a project of Austin Transportation's [Data & Technology Services](https://austinmobility.io/) team. Project status is documented regularly in the comments below.

**Comment by patrickm02L** [comment]
**Project Kickoff:** The project was initiated in March.

**Prototype Development**: The team has developed a prototype for the app using Knack.

**API Access Request:** The team has approached Passport for API access to plug into the Knack app. The aim is to get an accurate account of residents with active parking permits for their specific zone.

**Demo with Passport:** The team has demoed the prototype to Passport and discussed the integration with them.

**API Access Status:** Currently, the team is waiting on API access from Passport.

Overall, the project is moving ahead as planned, and the team has made significant progress in developing the prototype for the app. The next step is to obtain the necessary API access to continue with the integration process.

**Comment by patrickm02L** [comment]
On June 30, 2023, we delivered an external site that utilizes the Passport API to retrieve license plate information and determine their permitted status. 

On July 3, 2023, we had a meeting with Jacob C. to discuss the limitations of the API. It was revealed that the API does not provide a zone ID and corresponding color for the license plates. We informed Jacob about this issue, and he assured us that he would follow up with Passport to obtain the missing data.

**Comment by patrickm02L** [comment]
We have collaborated with the RPP team to understand how parking zones are defined and to align this information with data from the Passport API. Efforts are currently underway to coordinate with the Passport team to link their UUIDs to the zones established by RPP.

**Comment by patrickm02L** [comment]
We're continuing to work with Passport to access the zones through their API.

**Comment by patrickm02L** [comment]
This project has had any movement in the last 7 months, moving to Icebox.

**Comment by amenity** [comment]
@hlupico - any advice on what we should do with this project and its outstanding sub-issues?
